packages/preview/fletcher/0.4.4/docs/gallery/algebra-cube.typ

```diff
@@ -0,0 +1,47 @@
+#import "@preview/fletcher:0.4.2" as fletcher: diagram, node, edge
```

Comment:
There are a few imports in the examples that are still using some older versions.

Reply:
Thanks! I've just fixed these.